surveys/2023-annual-survey/questions.md

```diff
@@ -594,6 +594,21 @@ Aspects:
 >
 > This question gathers data on the communities perceptions of certain aspects of Rust at this point in time.
 
+### How do you build your Rust projects?
+
+Type: select one (optional)
+
+Aspects:
+
+- I use Cargo
+- I use some other build system
+- I combine Cargo and another build system
+- If you combine Cargo with (or just use) other build systems, which ones? (open response)
```

Comment:
```suggestion
- I just use Cargo
- I just use some another build system
- Please specify which build system are you using instead of (or with) Cargo (open response)
- ```

Having only one open option, I cannot make it clearer than this :)

Reply:
Having only one open option, I cannot make it clearer than this :)

Reply:
Since the question now allows selecting multiple answers, I think that `just` doesn't make sense here. If you use just Cargo, you'll click only on Cargo. If you use both Cargo and another build system, you check both.

There's a third option, and that's combining Cargo and another build system, which I would keep as an answer (`I combine Cargo and another build system`).

Reply:
Ok, I think I'm basically back to square one.

Reply:
Tbh I think that changing it to `all that apply` was all that was needed, in my opinion :sweat_smile: